### ISSUE TITLE
atmos machines ( scrubber, pump) that are already present in the map when turned off unwrenched and wrenched magically turn on on its own

### ISSUE THREAD

**Issue by Tlaltecuhtli** [tgstation/tgstation#37062]

**Comment by Jalleo** [comment]
Possibly a dupe report.

**Comment by ShizCalev** [comment]
i'm have a little difficulty understanding what you mean, can you rephrase it?

**Comment by SpaceManiac** [comment]
Caused by #36690. Unwrenching a pump of the `/on` type causes it to turn on automatically when re-wrenched.
